.drone.yml

```diff
@@ -1,59 +1,97 @@
 ---
-pipeline:
-  build:
-    privileged: true
-    image: rancher/dapper:1.11.2
-    volumes:
-    - /var/run/docker.sock:/var/run/docker.sock
-    commands:
-    - dapper ci
+kind: pipeline
+name: default
 
-  build-all-binaries:
-    privileged: true
-    image: rancher/dapper:1.11.2
-    volumes:
-    - /var/run/docker.sock:/var/run/docker.sock
-    environment:
-    - CROSS=1
-    commands:
-    - dapper build
-    when:
-      branch: [ master, release/* ]
-      event: tag
+platform:
+  os: linux
+  arch: amd64
 
-  stage-binaries:
-    image: rancher/dapper:1.11.2
-    commands:
-    - cp -r ./bin/* ./package/
-    when:
-      branch: [ master, release/* ]
-      event: tag
+steps:
+- name: ci
+  pull: default
+  image: rancher/dapper:1.11.2
+  commands:
+  - dapper ci
+  privileged: true
+  volumes:
+  - name: socket
+    path: /var/run/docker.sock
 
-  github_binary_prerelease:
-    image: plugins/github-release
-    prerelease: true
-    files:
-     - build/bin/rke_*
-     - dist/artifacts/${DRONE_TAG}/*
-    checksum:
-     - sha256
-    secrets: [github_token]
-    when:
-      branch: [ master, release/* ]
-      event: tag
-      ref:
-        include: [ refs/tags/*rc* ]
+- name: build-all-binaries
+  pull: default
+  image: rancher/dapper:1.11.2
+  commands:
+  - dapper build
+  environment:
+    CROSS: 1
+  privileged: true
+  volumes:
+  - name: socket
+    path: /var/run/docker.sock
+  when:
+    branch:
+    - master
```

Comment:
this doesn't work any more, you'll have to follow the rancher/rancher .drone.yml with refs.